run.sh

```diff
@@ -1,12 +1,21 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+
 export NEW_VERSION="${1}"
+export BRANCH_NAME="release-${1}"
 export GITHUB_ACTION_PATH="${2}"
 export RELEASE_BODY="$(awk -v version="${NEW_VERSION}" -f ${GITHUB_ACTION_PATH}/scripts/show-changelog.awk CHANGELOG.md)"
 
-node "${GITHUB_ACTION_PATH}/scripts/update-package-version.js"
-
 git config user.name github-actions
 git config user.email github-actions@github.com
 
+node "${GITHUB_ACTION_PATH}/scripts/update-package-version.js"
+
+git checkout -b "${BRANCH_NAME}"
+
 if ! (git add . && git commit -m "Release ${NEW_VERSION}" && git push);
 then
     echo "No changes"
```

Comment:
```suggestion
    echo "No changes"
    exit 1
```

Reply:
Maybe like this? https://github.com/MetaMask/action-monorepo-release-pr/blob/enable-hotfixes/src/create-release-pr.sh#L23-L29